.script/jsonFileValidator.ts

```diff
@@ -1,13 +1,26 @@
+import fs from "fs";
+import { SchemaError, Validator } from 'jsonschema';
 import { runCheckOverChangedFiles } from "./utils/changedFilesValidator";
 import { ExitCode } from "./utils/exitCode";
-import fs from "fs";
 import * as logger from "./utils/logger";
 
 export async function IsValidJsonFile(filePath: string): Promise<ExitCode> {
-  JSON.parse(fs.readFileSync(filePath, "utf8"));
+  const json = JSON.parse(fs.readFileSync(filePath, "utf8")); 
+  if (filePath.endsWith('WorkbooksMetadata.json')) {
+    validateWorkbookSchema(json);
+  }
   return ExitCode.SUCCESS;
 }
 
+function validateWorkbookSchema(workbooksMetadataJson: object) {
+  const schema = JSON.parse(fs.readFileSync('.script/utils/WorkbooksMetadataSchema.json', 'utf8'));
+  var validationResult = new Validator().validate(workbooksMetadataJson, schema);
+  if (!validationResult.valid) {
+    let errorMsg = validationResult.errors.map(err => err.message).join(", ");
+    throw new SchemaError(errorMsg, schema)
```

Comment:
What we will see in the logs? both errorMsg & schema or just errorMsg?

Reply:
just errorMsg.  for example:
- `additionalProperty "someIllegalProperty" exists in instance when not allowed`
- `requires property "workbookKey"`